Documents/Pseudocode/Pseudocode_Final

```diff
@@ -152,26 +152,26 @@ Class Network
 	
 	//takes a matrix object and the address to a function (returns a double, no arguments)
 	//assigns numbers as specified by the function passed
-	void randomizeMatrix(matrix<double> input_matrix, double (*distribution)(void))
+	void randomizeMatrix(matrix<double> input_matrix, double (*distribution)(void)) //I am still thinking about distribution(), need to look up for that [-Ami]
 	{
 		//Looping an index j from 1 to input_matrix.nr()
 			//Looping an index i from 1 to input_matrix.nc()
-				//input_matrix(i, j) = distribution();
+				//input_matrix(i, j) = distribution(); 
 	}
 
 	//forwardPropagation() is a function that sets all activation values for a single test data input. 
 	//It needs the layer of activations[0] to be assigned values from the test data.
 	void forwardPropagation(int mini_batch_index)
 	{
 		//activations[0] = (contents in file[mini_batch_index]) <-- can either do this elementwise once, or elementwise (to load an input vector), then using dlib's overriden = operator
-		//Looping an index i from 1 to num_layers
+		//Looping an index i from 1 to num_layers			// I would say to read from the file, use strtok to assign the values in the activations matrix and this has to be done element wise if we're using strtok. [-Ami]
```

Comment:
Yeah, hard to get around doing an element-by-element loop at least once during assignment, at least as far as I know

Reply:
Well, that was only for activations[0]!

Reply:
Luckily we don't need to assign the rest, then!